SConstruct

```diff
@@ -1681,10 +1682,13 @@ env['python_cmd_esc'] = quoted(env['python_cmd'])
 
 # Python Package Settings
 python_min_version = parse_version("3.8")
+# Newest Python version not supported/tested by Cantera
+python_max_p1_version = parse_version("3.12")
```

Comment:
I'm curious what this variable stands for.

Reply:
Max supported version plus 1 … tried to heed to your suggestion 😃